org.eclipse.xtext/src/org/eclipse/xtext/workspace/ProjectConfigProvider.java

```diff
@@ -12,15 +12,14 @@
 import org.eclipse.xtext.workspace.IProjectConfigProvider;
 import org.eclipse.xtext.workspace.ProjectConfigAdapter;
 
-@SuppressWarnings("all")
 public class ProjectConfigProvider implements IProjectConfigProvider {
-  @Override
-  public IProjectConfig getProjectConfig(final ResourceSet context) {
-    ProjectConfigAdapter _findInEmfObject = ProjectConfigAdapter.findInEmfObject(context);
-    IProjectConfig _projectConfig = null;
-    if (_findInEmfObject!=null) {
-      _projectConfig=_findInEmfObject.getProjectConfig();
-    }
-    return _projectConfig;
-  }
+	@Override
+	public IProjectConfig getProjectConfig(final ResourceSet context) {
+		ProjectConfigAdapter adapter = ProjectConfigAdapter.findInEmfObject(context);
+		IProjectConfig projectConfig = null;
+		if (adapter != null) {
+			projectConfig = adapter.getProjectConfig();
+		}
+		return projectConfig;
```

Comment:
I'd rewrite this to
```java
if (adapter != null) {
    return adapter.getProjectConfig();
}
return null;
```

Reply:
changed to `return adapter != null ? adapter.getProjectConfig() : null;`